hermes-console/static/partials/topic.html

```diff
@@ -136,10 +135,8 @@ <h3 class="panel-title">Message schema</h3>
 
     <div class="row" ng-show="config.messagePreviewEnabled">
         <div class="col-md-12">
-            <div class="panel panel-default">
-                <div style="position: relative">
```

Comment:
imho this additional is not necessary, we have one header, and one or two elements with `float: right`